toc/working-groups/app-runtime-interfaces.md

```diff
@@ -117,6 +117,7 @@ areas:
   - cloudfoundry/java-buildpack-system-test
   - cloudfoundry/java-test-applications
   - cloudfoundry/jvmkill
+  - cloudfoundry/bosh-package-java-release
```

Comment:
I'm not convinced `bosh-package-java-release` belongs to app runtime interfaces.

Reply:
`bosh-package-cf-cli-release` LGTM

Reply:
Ok, looks like all buildpacks are there, so I guess it's ok.